settings.gradle

```diff
@@ -1,7 +1,7 @@
 def eclipseFile = "eclipsePlugin/local.properties"
 if (new File(eclipseFile).exists()) {
-  include ':spotbugsTestCases', ':spotbugs', ':eclipsePlugin', ':test-harness', ':spotbugs-annotations', ':spotbugs-ant'
+  include ':eclipsePlugin', ':findbugsTestCases', ':findbugs', ':gradlePlugin', ':test-harness', ':spotbugs-annotations', ':spotbugs-ant', 'gradlePlugin'
```

Comment:
Use `':spotbugsTestCases', ':spotbugs'` instead of `':findbugsTestCases', ':findbugs'`

Recently we changed name of directories at #141 and #137.